File: editors/chexedit/files/patch-src-init.c
--- src/init.c.orig	1999-08-08 05:56:11.000000000 +0400
+++ src/init.c	2015-01-12 20:23:12.000000000 +0300
@@ -250,6 +250,7 @@
    if (color_term != -1)
    {
       start_color ();
+      use_default_colors();
       if ((color_term = has_colors ()))
       {
          color_term = 1;

File: devel/phpsh/files/patch-src____init__.py
--- ./src/__init__.py.orig	2014-07-08 19:24:25.000000000 +0800
+++ ./src/__init__.py	2014-07-08 19:24:45.000000000 +0800
@@ -758,7 +758,7 @@
                 ret_code = self.p.poll()
                 if debug:
                     print "ret_code: " + str(ret_code)
-                if ret_code != None:
+                if ret_code not in (None, 0):
                     if debug:
                         print "NOOOOO"
                     print "subprocess died with return code: " + repr(ret_code)
